tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/io/graphson/AbstractTinkerGraphGraphSONTranslatorProvider.java

```diff
@@ -238,8 +228,7 @@ public abstract class AbstractTinkerGraphGraphSONTranslatorProvider extends Tink
 
     @Override
     public GraphTraversalSource traversal(final Graph graph) {
-        final GraphTraversalSource g = graph.traversal();
-        return g.withStrategies(new TranslationStrategy(g, new GraphSONTranslator<>(JavaTranslator.of(g), version), true));
+        return graph.traversal();
```

Comment:
similar question here as asked earlier - what are we really testing here after this change?